libcudacxx/include/cuda/std/detail/libcxx/include/version

```diff
@@ -219,6 +219,9 @@ __cpp_lib_void_t                                        201411L <type_traits>
 #pragma GCC system_header
 #endif
 
+// We define certain feature test macros depending on availability
+# define __cccl_lib_local_barrier_arrive_tx
+
```

Comment:
This feature test macro should be guarded based on architecture similar to the `barrier` header itself. 
```suggestion
// We define certain feature test macros depending on availability
#if defined(__CUDA_MINIMUM_ARCH__) && __CUDA_MINIMUM_ARCH__ < 700
# define __cccl_lib_local_barrier_arrive_tx
#endif
```

Reply:
Nvm. I the feature flag test can be wrapped in an ifdef as well: https://github.com/NVIDIA/cccl/pull/358/commits/e466daa9492221511b2b82073dc193fedbbb64da